GitTabExpansion.ps1

```diff
@@ -74,6 +74,12 @@ function script:gitBranches($filter, $includeHEAD = $false) {
         foreach { $prefix + $_ }
 }
 
+function script:gitTags($filter) {
+    git tag |
+        where { $_ -like "$filter*" } |
+        foreach { $_ }

```

Comment:
It provides tab completion based on tag names and is called from GitTabExpansion().